setup.cfg

```diff
@@ -39,17 +39,12 @@ exclude = extern,sphinx,*parsetab.py
 [metadata]
 package_name = ndcube
 description = A base package for multi-dimensional contiguious and non-contiguious spatially aware arrays.
-long_description = 
 author = SunPy Developers
 author_email = sunpy@googlegroups.com
 license = BSD-2-Clause
 url = http://sunpy.org
 edit_on_github = True
 github_project = sunpy/ndcube
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-version = 0.1.dev
-
-[entry_points]
-
-# astropy-package-template-example = packagename.example_mod:main
+version = 1.0.dev
```

Comment:
Should version just be `1.0`?  Not `1.0.dev`?

Reply:
I will bump to 1.0 later in the release process.